axltempl/drupal.py

```diff
@@ -133,6 +133,11 @@ def main(
         util.write_info("Adding GitLab support...")
         gitlab.generate_gitlab_files(docroot)
 
+    if not no_install:
+        run_composer_install()
```

Comment:
@skippednote, reordering won't work because it is `composer install` that brings in the settings file.